devfiles/simple-devfile.yaml

```diff
@@ -4,11 +4,25 @@ projects:
         location: "https://github.com/che-incubator/devworkspace-api"
         branch: "master"
 commands:
-  - name: Create json schema from Go API
-    actions:
-      - type: exec
-        command: "./buildSchema.sh"
-        component: build-tools
+  - exec:
+      commandLine: "./buildSchema.sh"
+      component: build-tools
```

Comment:
Well, a component may have multiple containers.
Maybe it makes sense to consider improving format to solve this issue. Maybe it would be better/easier at this stage just to postpone it.